primitives/state-machine/src/ext.rs

```diff
@@ -152,6 +157,19 @@ where
 	B: 'a + Backend<H>,
 	N: crate::changes_trie::BlockNumber,
 {
+
+	fn set_offchain_storage(&mut self, key: &[u8], value: Option<&[u8]>) {
+		// write to storage kind `StorageKind::PERSISTENT`````` which
+		// is in turn mapped to `STORAGE_PREFIX` .
+		// MUST be kept in sync with  `sp_offchain::STORAGE_PREFIX`
+		// which can not be used directly, since it causes a circular dependency.
+		const STORAGE_PREFIX : &'static [u8] = b"storage";
```

Comment:
Is there a way to test that these are still in-sync? Maybe a unit-test in `sp_offchain`?